### DISCUSSION TITLE
Bug in auth hook password verification example

### DISCUSSION THREAD

**Discussion by florianjuengermann** [supabase/supabase#36088] [Troubleshooting]
# Improve documentation

## Link 

https://supabase.com/docs/guides/auth/auth-hooks/password-verification-hook

## Describe the problem

The example policy does not work as intended. It does not lock any users out; you can always log in with the correct password. 

## Describe the improvement

The correctness check for the password needs to come after the lockout check.

## Additional context

See my comment at https://github.com/supabase/supabase/pull/25880

**Comment by charislam** [comment]
@J0 tagging you for triage since you wrote the auth hooks docs 🙏🏼
